### DISCUSSION TITLE
Is funnel analysis feature a supported in self-hosted?

### DISCUSSION THREAD

**Discussion by nnthuanegany** [plausible/analytics#3442] [Self-Hosted Support]
In doc: https://plausible.io/docs/funnel-analysis i see

"Funnels is an upcoming premium functionality that's free-to-use during the private preview. Pricing will be announced soon. "

I'm not sure if channel analytics is supported in self-hosted?

So I'd really thanks someone confirming this information for me ^^

**Comment by ruslandoga** [comment]
👋 @nnthuanegany 

The current latest release [`v2.0.0`](https://github.com/plausible/analytics/discussions/3132) doesn't include funnels.

**Reply by nnthuanegany** [reply]
👋 @ruslandoga thank you reply,

So do you know when this feature will be ready for the self-hosted version?

**Reply by nnthuanegany** [reply]
I see info with self-hosted

"[It's a long term release](https://plausible.io/blog/building-open-source) published twice per year. Selected premium features are not available as we aim to ensure a protective barrier around our cloud offering."

and the funnel is premium function, so it will not include in self-hosted version?

**Reply by alekseyp** [reply]
https://github.com/plausible/analytics/discussions/3099

**Reply by nnthuanegany** [reply]
@ruslandoga Can you confirm? This is very important for me to choose my solution. Thanks a lot.

> 
> I see info with self-hosted
> 
> "[It's a long term release](https://plausible.io/blog/building-open-source) published twice per year. Selected premium features are not available as we aim to ensure a protective barrier around our cloud offering."
> 
> and the funnel is premium function, so it will not include in self-hosted version?
